tgui/packages/tgui/interfaces/SecurityConsole.js

```diff
@@ -0,0 +1,534 @@
+import { Box, Button, Section, Table, Input, NoticeBox, Icon, Fragment, NumberInput, LabeledList, Flex } from '../components';
+import { useBackend, useLocalState } from '../backend';
+import { Window } from '../layouts';
+import { FlexItem } from '../components/Flex';
+import { TableRow, TableCell } from '../components/Table';
+
+export const SecurityConsole = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    logged_in,
+    username,
+    has_access,
+    active_general_record,
+    min_age,
+    max_age,
+    screen } = data;
+  const { theme = 'ntos' } = props;
+
+  const [searchText, setSearchText] = useLocalState(context, "searchText", "");
+
+  if (data.z > 6) {
+    return (
+      <Window resizable theme={theme}>
+        <Window.Content scrollable>
+          <NoticeBox>
+            Unable to establish connection. You are too far away from the station!
+          </NoticeBox>
+        </Window.Content>
+      </Window>
+    );
+  }
+
+  if (!logged_in) {
+    return (
+      <Window resizable theme={theme}>
+        <Window.Content scrollable>
+          <Section title="Welcome">
+            <Flex align="center" justify="center" mt="0.5rem">
+              <Flex.Item>
+                <Fragment>
+                  {data.user_image && (
+                    <img src={"photo_" + data.user_image +"_cropped.png"}
+                      width="125px" height="125px"
+                      style={`-ms-interpolation-mode: nearest-neighbor;
+                      border-radius: 50%; border: 3px solid white;`} />
```

Comment:
```suggestion
                      border-radius: 50%; border: 3px solid white;background-color:black;`} />
```
Or whatever colour you want. Just please add the background. Probably try a blue that's slightly darker than the rest of the menu
![image](https://user-images.githubusercontent.com/20558591/87480285-4c0d7600-c62d-11ea-8233-35085059f64e.png)